nats-concepts/jetstream/consumers.md

```diff
@@ -1,234 +1,94 @@
-The consumer's sample frequency _can_ be adjusted after the initial creation of the consumer
-{% endhint %}
-
-## Pull consumer specific attributes
-These attributes apply only to pull consumers.
-
-### MaxWaiting
-
-The maximum number of waiting pull requests.
-
-{% hint style="warning" %}
-The consumer's maximum number of waiting pull requests **_can not_** be adjusted after the initial creation of the consumer
-{% endhint %}
-
-## Push consumer specific attributes
-These attributes apply only to push consumers.
-
-### DeliverSubject
-
-The subject to deliver observed messages, specifying a delivery subject makes the consumer a 'push consumer' as 'pull consumers' do not need a static delivery subject. If you want to distribute the messages between the subscribers to the consumer then you also need to specify a queue group name.
+If an ack is required but is not received within the `AckWait` window, the message will be redelivered
 
 {% hint style="warning" %}
-The push consumer's deliver subject **_can not_** be adjusted after the initial creation of the consumer
+The server may consider an ack arriving out of the window. If a first process fails to ack within the window it's entirely possible, for instance in queue situation, that the message has been redelivered to another consumer. Since this will technically restart the window, the ack from the first consumer will be considered.
 {% endhint %}
 
-### DeliverGroup
+### DeliverPolicy
 
-The queue group name which, if specified, is then used to distribute the messages between the subscribers to the consumer.
+The policies choices include:
 
-{% hint style="info" %}
-The push consumer's queue group name _can_ be adjusted after the initial creation of the consumer
-{% endhint %}
+- `DeliverAll` -  The default policy. The consumer will start receiving from the earliest available message.
+- `DeliverLast` - When first consuming messages, the consumer will start receiving messages with the last message added to the stream, or the last message in the stream that matches the consumer's filter subject if defined.
+- `DeliverLastPerSubject` - When first consuming messages, start with the latest one for each filtered subject currently in the stream.
+- `DeliverNew` - When first consuming messages, the consumer will only start receiving messages that were created after the consumer was created.
+- `DeliverByStartSequence` - When first consuming messages, start at this particular message in the set. The consumer is required to specify `OptStartSeq`, the sequence number to start on. It will receive the closest available message moving forward in the sequence should the message specified have been removed based on the stream limit policy.
+- `DeliverByStartTime` - When first consuming messages, start with messages on or after this time. The consumer is required to specify `OptStartTime`, the time in the stream to start at. It will receive the closest available message on or after that time.
```

Comment:
I know I'm the grammar police here but can we reword this one "The consumer is required to specify...., the time in the stream to start at." to '....the time at which the stream should start.' ?